dist/src/test/java/io/camunda/application/commons/backup/BackupPrioritiesTest.java

```diff
@@ -120,6 +120,96 @@ public void testBackupPriorities(
+
+    final var indices = priorities.indicesSplitBySnapshot().toList();
+
+    assertThat(indices.size()).isEqualTo(8);
+    // PRIO 1
+    assertThat(indices.get(0).indices())
+        .containsExactlyInAnyOrder(
+            "operate-import-position-8.3.0_",
+            "tasklist-import-position-8.2.0_"); // ADD optimize-position-based-import-index_v3)
+    // PRIO 2
+    assertThat(indices.get(1).indices())
+        .containsExactlyInAnyOrder("operate-list-view-8.3.0_", "tasklist-task-8.5.0_");
+    // PRIO 2 TEMPLATES
+    assertThat(indices.get(2).indices())
+        .containsExactlyInAnyOrder(
+            "operate-list-view-8.3.0_*",
+            "-operate-list-view-8.3.0_",
+            "tasklist-task-8.5.0_*",
+            "-tasklist-task-8.5.0_");
+    // PRIO 3
+    assertThat(indices.get(3).indices())
+        .containsExactlyInAnyOrder("operate-batch-operation-1.0.0_", "operate-operation-8.4.1_");
+    // PRIO 4
+    assertThat(indices.get(4).indices())
+        .containsExactlyInAnyOrder(
+            "operate-decision-8.3.0_",
+            "operate-decision-instance-8.3.0_",
+            "operate-event-8.3.0_",
+            "operate-flownode-instance-8.3.1_",
+            "operate-job-8.6.0_",
+            "operate-incident-8.3.1_",
+            "operate-message-8.5.0_",
+            "operate-post-importer-queue-8.3.0_",
+            "operate-sequence-flow-8.3.0_",
+            "operate-user-task-8.5.0_",
+            "operate-variable-8.3.0_",
+            "tasklist-draft-task-variable-8.3.0_",
+            "tasklist-task-variable-8.3.0_");
+
+    // PRIO 4 TEMPLATES
+    assertThat(indices.get(5).indices())
+        .containsExactlyInAnyOrder(
+            "operate-event-8.3.0_*",
+            "-operate-event-8.3.0_",
+            "operate-flownode-instance-8.3.1_*",
+            "-operate-flownode-instance-8.3.1_",
+            "operate-incident-8.3.1_*",
+            "-operate-incident-8.3.1_",
+            "operate-job-8.6.0_*",
+            "-operate-job-8.6.0_",
+            "operate-message-8.5.0_*",
+            "-operate-message-8.5.0_",
+            "operate-post-importer-queue-8.3.0_*",
+            "-operate-post-importer-queue-8.3.0_",
+            "operate-sequence-flow-8.3.0_*",
+            "-operate-sequence-flow-8.3.0_",
+            "operate-user-task-8.5.0_*",
+            "-operate-user-task-8.5.0_",
+            "operate-variable-8.3.0_*",
+            "-operate-variable-8.3.0_",
+            "-operate-decision-instance-8.3.0_",
+            "operate-decision-instance-8.3.0_*",
+            "-tasklist-draft-task-variable-8.3.0_",
+            "tasklist-draft-task-variable-8.3.0_*",
+            "-tasklist-task-variable-8.3.0_",
+            "tasklist-task-variable-8.3.0_*");
+
+    // PRIO 5
+    assertThat(indices.get(6).indices())
+        .containsExactlyInAnyOrder(
+            //            "operate-web-session-1.1.0_", // TODO Does it exist?
```

Comment:
can we remove the TODO?

Reply:
I did not get a reply yet on the thread, that's why I didn't remove it :+1: